package.json

```diff
@@ -38,7 +40,9 @@
     "image"
   ],
   "dependencies": {
-    "prop-types": "^15.5.8"
+    "prop-types": "^15.5.8",
+    "react": "^15.5.4",
```

Comment:
No, I think we should keep that, we cannot rely on that being provided by the using app.